src/lib/docs.ts

```diff
@@ -1,4 +1,4 @@
-import {relative, resolve} from 'path';
+import {posix} from 'path';
```

Comment:
Not sure why we need `posix`, for windows?

Reply:
yep, path delimiter on windows is `\\` and breaks tests. using posix solves cross-platform.

Reply:
Gotcha, sounds good then. Thanks!